src/components/Block/index.js

```diff
@@ -1,104 +1,100 @@
+${({ left }) => (left !== undefined) && `left: ${(left)};`}
+${({ right }) => (right !== undefined) && `right: ${(right)};`}
+${({ bottom }) => (bottom !== undefined) && `bottom: ${(bottom)};`}
 `;
 
 const superProps = separateLiteralToProps`
-	${({ relative }) => relative && 'position: relative;'}
-	${({ absolute }) => absolute && 'position: absolute;'}
-	${({ centred }) => centred && 'justify-content: center; align-items: center;'}
-	${({ js }) => js && 'justify-content: flex-start;'}
-	${({ jc }) => jc && 'justify-content: center;'}
-	${({ je }) => je && 'justify-content: flex-end;'}
-	${({ jb }) => jb && 'justify-content: space-between;'}
-	${({ as }) => as && 'align-items: flex-start;'}
-	${({ ac }) => ac && 'align-items: center;'}
-	${({ ae }) => ae && 'align-items: flex-end;'}
-	${({ centred }) => centred && 'justify-content: center; align-items: center;'}
-	${({ expand, full, filled }) => (expand || full || filled) && 'width: 100%; height: 100%'}
-	${({ flexed }) => flexed && 'flex: 1'}
-	${({ row }) => row && 'flex-direction: row'}
-	${({ color }) => color && `background-color: ${color};`}
-	${({ white }) => white && 'background-color: white'}
-	${({ hidden }) => hidden && 'overflow: hidden;'}
-	${({ black }) => black && 'background-color: black'}
-	${({ red }) => red && 'background-color: red'}
-	${({ blue }) => blue && 'background-color: blue'}
-	${({ green }) => green && 'background-color: green'}
-	${({ yellow }) => yellow && 'background-color: yellow'}
-	${({ oWhite }) => oWhite && 'background-color: white; opacity: 0.5'}
-	${({ oBlack }) => oBlack && 'background-color: black; opacity: 0.5'}
-	${({ oRed }) => oRed && 'background-color: red; opacity: 0.5'}
-	${({ oBlue }) => oBlue && 'background-color: blue; opacity: 0.5'}
-	${({ oGreen }) => oGreen && 'background-color: green; opacity: 0.5'}
-	${({ oYellow }) => oYellow && 'background-color: yellow; opacity: 0.5'}
+${({ relative }) => relative && 'position: relative;'}
+${({ absolute }) => absolute && 'position: absolute;'}
+${({ centred }) => centred && 'justify-content: center; align-items: center;'}
+${({ js }) => js && 'justify-content: flex-start;'}
+${({ jc }) => jc && 'justify-content: center;'}
+${({ je }) => je && 'justify-content: flex-end;'}
+${({ jb }) => jb && 'justify-content: space-between;'}
+${({ as }) => as && 'align-items: flex-start;'}
+${({ ac }) => ac && 'align-items: center;'}
+${({ ae }) => ae && 'align-items: flex-end;'}
+${({ centred }) => centred && 'justify-content: center; align-items: center;'}
+${({ expand, full, filled }) => (expand || full || filled) && 'width: 100%; height: 100%'}
+${({ flexed }) => flexed && 'flex: 1'}
+${({ row }) => row && 'flex-direction: row'}
+${({ color }) => color && `background-color: ${color};`}
+${({ white }) => white && 'background-color: white'}
+${({ hidden }) => hidden && 'overflow: hidden;'}
+${({ black }) => black && 'background-color: black'}
+${({ red }) => red && 'background-color: red'}
```

Comment:
Is it necessary after` ${({ color }) => color && `background-color: ${color};`}` ?